cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -1483,7 +1484,10 @@ runTransactionSubmitCmd
 
     res <- liftIO $ submitTxToNodeLocal localNodeConnInfo txInMode
     case res of
-      Net.Tx.SubmitSuccess -> liftIO $ Text.putStrLn "Transaction successfully submitted."
+      Net.Tx.SubmitSuccess -> do
+        liftIO $ Text.putStrLn "Transaction successfully submitted. Transaction hash is:"
+        let hashText = T.decodeUtf8 $ serialiseToRawBytesHex (getTxId $ getTxBody tx)
```

Comment:
This duplicates the `ToJSON TxId` instance. Can we avoid that?

Reply:
Changed to use `TxId`'s JSON format :+1: